packages/pieces/community/legis/src/lib/actions/create-ticket.ts

```diff
@@ -294,7 +295,8 @@ export const createTicket = createAction({
                     limitDay = subtractDays(new Date(dateCurrent), days);
                   }
               }
-              body['data_hora'] = limitDay.toISOString();
+              body['data_hora'] = limitDay.toISOString().split(".")[0]
```

Comment:
Adicionar `optional chaining`
```suggestion
              body['data_hora'] = limitDay.toISOString()?.split(".")?.[0]
```